### ISSUE TITLE
(8) Ingest Lange (MS 20)

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#440]
### Descriptive Summary

Series of complex image objects belonging to the Ruth-Marion Baruch and Pirkle Jones Photographs Collection (MS 20).

### Background 

This is a series that was scanned by Pixel Acuity, but metadata issues prevented us from ingesting it with the rest of MS 20.

### Acceptance Criteria

This is what done looks like:

* [x] Files and metadata have been successfully ingested
* [x] Files and metadata have been added as members of the Ruth-Marion Baruch and Pirkle Jones Photographs Collection (MS 20)
* [x] Collection has be QA'd (images display properly, metadata displays properly, etc.)
* [ ] Collection has been published and is available to users

### Related Work 

Pre-Ingest Dorothea Lange (MS 20) #439

**Comment by rmjaffe** [comment]
Status update: Lange images have been ingested (included as part of the greater MS 20 collection) but are presently unpublished in the DAMS.  There is an issue with the values in the creator metadata property.  The URI supplied in the ingest spreadsheet (https://id.loc.gov/authorities/names/n79100159) is displaying on the metadata records, not the label 'Lange, Dorothea'. 

<img width="558" alt="Screen Shot 2021-06-24 at 8 04 43 AM" src="https://user-images.githubusercontent.com/32003407/123311485-f25f2e00-d4c2-11eb-829a-f18c4247619d.png">

Example: https://digitalcollections.library.ucsc.edu/records/tm70n0134

This is true for all items included in the ingest; and will need to be corrected.  The only way to isolate this MS 20 ingest on the front end is to search by the URI (https://id.loc.gov/authorities/names/n79100159).

**Comment by rmjaffe** [comment]
Per Sue, "We did do some light QA on Lange. There were no issues with the ingest of the images and metadata.  The only problem was that the system is displaying the LOC URI for Lange's name instead of the the text "Dorothea Lange"."  Publication of this collection is on hold until External controlled vocabularies not loading when editing metadata records #462 is resolved.

**Comment by snehagunduraoUL** [comment]
Collection is displayed as expected. Ticket can be closed.
<img width="1607" alt="Screen Shot 2021-12-08 at 1 09 43 PM" src="https://user-images.githubusercontent.com/92340457/145285066-c5a214c6-a175-472e-bd08-074568f00a18.png">

**Comment by rmjaffe** [comment]
@snehagunduraoUL Have the Lange images been published?  Are they displaying to endusers as part of the Ruth-Marion Baruch & Pirkle Jones Collection?  I'm only seeing them when I am logged in.

**Comment by snehagunduraoUL** [comment]
@rmjaffe yes..looks like it.
<img width="608" alt="Screen Shot 2021-12-15 at 10 46 21 AM" src="https://user-images.githubusercontent.com/92340457/146246706-494935dc-46a6-4f4f-8e21-3aaa5659a573.png">

**Comment by rmjaffe** [comment]
Yes, this is one of the images that was loaded and published as part of an earlier MS 20: Ruth-Marion Baruch and Pirkle Jones ingest.  The creator value on all the items that went in as part of the more recent Lange addition to MS 20 in "Lange, Dorothea" (https://id.loc.gov/authorities/names/n79100159).  While signed in I can search by the URI to view the Lange images, but when logged out, I'm unable to view them.  

@tmariamora:  Do we want to move ahead in making the Lange image publicly available or do we want to wait until other development work has been resolved or until SC&A has bandwidth to promote them?

**Comment by tmariamora** [comment]
@rmjaffe I think it depends on what additional development work may need to be done. I'm fine to move forward with publishing but would want a bit of lead time so we can communicate with OMCA and put together a press release.

**Comment by rmjaffe** [comment]
@tmariamora I'm thinking let's finish the bulkrax implementation/Scooby removal first, and then as we QA the published DAMS collections post-migration, let's check back in about this subset.
